### PR TITLE
Various css fixes

### PR DESCRIPTION
**Pull request by melissasamworth**

### CODE REVIEW

File: app/components/ArticlesDropdown/index.tsx [hunk 1]
```diff
@@ -28,9 +28,17 @@ export const ArticlesDropdown = ({toc, categories}: ArticlesDropdownProps) => {
...
+  const ArticlesSection = ({
+    category,
+    toc,
+    className,
+  }: {
+    category: Category
+    toc: TOCItem[]
+    className?: string
+  }) => (
+    <div className={'myClassName ' + (className || '')}>
```
Comment (mruwnik) [inline app/components/ArticlesDropdown/index.tsx:40]: should this be `myClassName`?

File: app/components/Nav/nav.css [hunk 2]
```diff
@@ -16,9 +16,14 @@
   width: 100%;
   height: 100%;
 }
+
+.top-logo {
+  padding: 0 56px 0 0;
```
Comment (mruwnik) [inline app/components/Nav/nav.css:21]: what happened to `var(--spacing-56)`?

File: app/components/Nav/nav.css [hunk 3]
```diff
@@ -16,9 +16,14 @@
...
+}
+
 .top-menu {
   display: flex;
-  gap: 20px;
+  gap: 40px;
```
Comment (mruwnik) [inline app/components/Nav/nav.css:26]: `var(--spacing-40)`?

File: app/newRoot.css [hunk 4]
```diff
@@ -113,18 +115,20 @@ h2 {
...
+  font-size: 14px;
   letter-spacing: -0.14px;
   font-weight: 600;
 }
 
 /* color classes */
 
 .black {
-  color: var(--colors-black, #000);
+  color: var(--text-black);
```
Comment (mruwnik) [inline app/newRoot.css:131]: is `--text-black` defined? I can't see it

**Review by mruwnik — APPROVED** [review]

File: app/components/ArticlesDropdown/index.tsx (hunk 1 above)
Comment (melissasamworth) [reply app/components/ArticlesDropdown/index.tsx:40]: Right, no classes need to be applied to component at large so I removed 'myClassName ' + , hope that is the right move.

File: app/newRoot.css (hunk 4 above)
Comment (melissasamworth) [reply app/newRoot.css:131]: I don't remember adding this line, weird. Fixed now.

**Pull request merged**